src/bci_build/package/appcontainers.py

```diff
@@ -86,6 +86,8 @@ def _envsubst_pkg_name(os_version: OsVersion) -> str:
 # This can be removed after the pcp dependency on sysconfig is removed
 {DOCKERFILE_RUN} systemctl disable wicked wickedd || :
 
+{DOCKERFILE_RUN} useradd --comment "PCP Quality Assurance" --create-home --home-dir /var/lib/pcp/testsuite --shell /bin/bash --uid {"483" if os_version.is_sle15 else "496"} pcpqa
```

Comment:
this user is only used for testing it seems. so can we just always use 496 rather than having a sle15 specific uid?

Reply:
we only need to keep it stable for stuff that is overwritten via configuration or volume mounts, which doesn't seem to be the case here?